tests/integration/components/bs-alert-test.js

```diff
@@ -1,6 +1,7 @@
 import { module } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, click, waitFor } from '@ember/test-helpers';
+import { set } from '@ember/object';
```

Comment:
Any reason to use the imported `set()` here? I understand using it helps for objects that you don't know if they extend from `Ember.Object`, i.e. have a `.set()`. But tbh I am not a fan of `set(this, ...)` when we *do know*  what `this` is. So I'd prefer just using `this.set()` below, as this would be also more consistent with the rest of the codebase.

Reply:
I added `set()` by habit, heh.

In the tests, I initialized `this.isAlertShown` to `true`. I think, if we subsequently write `this.isAlertShown = false` instead of `set(this, 'isAlertShown', false)`, we get an error?

I haven't checked if we still get the error nowadays.

Reply:
Changed syntax to `this.set()`